qutebrowser/keyinput/keyutils.py

```diff
@@ -298,22 +314,25 @@ class KeyInfo:
     Attributes:
         key: A Qt::Key member.
         modifiers: A Qt::KeyboardModifiers enum value.
+        text: A QString with the value of QKeyEvent::text()
     """
 
     key = attr.ib()
     modifiers = attr.ib()
 
     @classmethod
     def from_event(cls, e):
-        return cls(e.key(), e.modifiers())
+        return cls(_remap_unicode(e.key(), e.text()), e.modifiers())
 
     def __str__(self):
         """Convert this KeyInfo to a meaningful name.
 
         Return:
             A name of the key (combination) as a string.
         """
+
         key_string = _key_to_string(self.key)
+
```

Comment:
Don't see why those two whitespace changes are needed (and the first one triggers flake8).

Reply:
Ah, those came from an if statement in the previous approach. Tidied.